spack-repo/packages/singularity-eos/package.py

```diff
@@ -63,6 +63,7 @@ class SingularityEos(CMakePackage, CudaPackage):
 
     depends_on("eospac", when="+eospac")
     depends_on("spiner")
+    depends_on("ports-of-call@1.4.0")
```

Comment:
Should be
```suggestion
    depends_on("ports-of-call@1.4.0:")
```
You also want the same in `spiner`.

Reply:
Spiner isn't using the new code. I'll add the colon in singularity.